src/app/organisation/index.njk

```diff
@@ -65,16 +80,48 @@
             $("#organisation_table_row tr").filter(function () {
```

Comment:
There are 2 separate script tag (not including the dynamic one).
* They should be in a separate file or at least the same script tag.

Reply:
done.  removed all js code from organisation nunjukes file and put it in the new separate js file - 'src/public/javascript/organisation.js'